src/content/lodash/lang/isArrayLike/vanilla.js

```diff
@@ -0,0 +1,19 @@
+const isArrayLike = a =>
+  typeof a !== 'function' &&
+  a.length >= 0 &&
+  a.length <= Number.MAX_SAFE_INTEGER
```

Comment:
This implementation will throw an error if `a` will be `null`, `undefined`, etc. (Due to this, it also treats `document.all` correctly -- you can iterate over it, but `_.isArrayLike` returns `false`, since `document.all == null`)

Reply:
right, the fix should be trivial, thanks for spotting it :)

Reply:
Basically something like this will work perfectly both with falsy values (and with `document.all`, which might be kind of ok, even if it is a discrepancy comparing to lodash)

```suggestion
const isArrayLike = a =>
  typeof a !== 'function' &&
  a?.length >= 0 &&
  a?.length <= Number.MAX_SAFE_INTEGER
```

Reply:
LGTM, I will commit this and then add to the resources a reference to the optional chaining

Reply:
> LGTM, I will commit this and then add to the resources a reference to the optional chaining

FYI: I've created a PR to `lodash` to address the `_.isArrayLike(document.all)` 🙂  https://github.com/lodash/lodash/pull/5617